Loop/Managers/LoopDataManager.swift

```diff
@@ -1676,27 +1682,6 @@ extension LoopDataManager {
             }
 
             dosingDecision.automaticDoseRecommendation = recommendedAutomaticDose?.recommendation
-
-            let volumeRounder = { (_ units: Double) in
-                return self.delegate?.loopDataManager(self, roundBolusVolume: units) ?? units
-            }
-
-            let pendingInsulin = try getPendingInsulin()
-            let predictionDrivingBolusRecommendation = pendingInsulin > 0 ? predictedGlucoseIncludingPendingInsulin : predictedGlucose
-            let recommendation = predictionDrivingBolusRecommendation.recommendedManualBolus(
-                to: glucoseTargetRange!,
-                at: predictedGlucose[0].startDate,
-                suspendThreshold: settings.suspendThreshold?.quantity,
-                sensitivity: insulinSensitivity!,
-                model: doseStore.insulinModelProvider.model(for: pumpInsulinType),
-                pendingInsulin: 0, // Pending insulin is already reflected in the prediction
-                maxBolus: maxBolus!,
-                volumeRounder: volumeRounder
-            )
-            recommendedManualBolus = (recommendation: recommendation, date: startDate)
-            self.logger.debug("Recommending bolus: %{public}@", String(describing: recommendedManualBolus))
-
-            dosingDecision.manualBolusRecommendation = ManualBolusRecommendationWithDate(recommendedManualBolus)
```

Comment:
If you are removing the `manualBolusRecommendation` from a "loop", then I think you can also get rid of lines 852-853 in the `loop` function:
```
// The manual bolus recommendation is ignored for looping, so clear
dosingDecision.manualBolusRecommendation = nil
```
These lines are there only to "cleanup" a loop dosing decision by removing the extraneous `manualBolusRecommendation`.